### ISSUE TITLE
Multipler's on items depending on Raid instance

### ISSUE THREAD

**Issue by krisis6** [Roeshambo/MonolithDKP#15]
**A multiplier for raids would be good...,  so you dont have to manually change your prices depending on raid you are in.**

So you could set 100 for head slot in mc and have a multipler for each raid as you set the min or exact bid price

and BWL you set a multipler of 20% increase.. AQ 40%increase Naxx 100% increase
so effectively depending on raid:

it would look like this
MC Head drops = 100dkp
BWL head drops = 120 DKP
AQ Head drops 140dkp
Naxx head drops 200dkp

So just by having a minimum / exact bid price, but depending on which instance you are in it'll set to the appropriate multipler...

**So tldr. you set your prices like you have, but multiplier option for all raids**
MC
ZG
NAX
BWL
AQ20
AQ40 
World boss
etc...

**Comment by Roeshambo** [comment]
Can look into this
